test/functionalTest/cases/1177_geometry_and_coords/geometry_and_coords_errors.test

```diff
@@ -119,85 +119,85 @@ Date: REGEX(.*)
 
 02. GET /v2/entities with erroneous areaType - ERROR
 ====================================================
-HTTP/1.1 422 Unprocessable Entity
-Content-Length: 135
+HTTP/1.1 400 Bad Request
+Content-Length: 118
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "description": "Query not supported: error parsing geometry: Invalid selector in geometry specification",
-    "error": "InvalidModification"
+    "description": "Wrong query: error parsing geometry: Invalid selector in geometry specification",
```

Comment:
This change in .test (and some other similar) corresponds to the following entry in CNR: "NGSIv2 geoquery syntax errors using 400 Bad Request instead of 422 Unprocessable Entity (according to rules defined in #1286)".

Note that specification says:

> If an implementation is not able to resolve a geographical query, the HTTP Status code of the response must be 422, Unprocessable Entity. The error name, present in the error payload, must be NotSupportedQuery. 

However, this case correspond to a query that *being correct from a sintax point of view* (i.e. not the case in this .test) cannot be processed by the NGSIv2 server implementation due to some limitation (e.g. limitations in the DB engine to solve a particular geometric shape).

NTC (just a remark).